usim/_primitives/timing.py

```diff
@@ -1,8 +1,10 @@
 """
-There is no inherent time unit, such as seconds, hours, or years, implied for simulation time.
+There is no inherent time unit, such as seconds, hours, or years,
+implied for simulation time.
 A simulation should use a consistent time unit, however.
 
-The default time type is :py:class:`float`, which in principle may exhibit imprecision for fractions.
+The default time type is :py:class:`float`, which in principle may exhibit
```

Comment:
I am struggling with *in principle*. We might write "which in theory may" or "which might".